### PR TITLE
[DESIGN2-205-Dropdown] Dropdown Component - DSCO - Update color variables

### PR DESCRIPTION
**Pull request by kelbyhawn**
Swaps out color variables from `shared/color.scss` with Primitive colors from `primitiveColors.css` on the Dropdown component.

👀 **DOTD note:** there might be Eyes diffs with this.

## Links
Jira ticket: [DESIGN2-205](https://codedotorg.atlassian.net/browse/DESIGN2-205)

## Testing story
Tested locally in Storybook

----

### Before - using `shared/colors.scss`

https://github.com/user-attachments/assets/414fa263-4c9f-445e-8f73-abd295ec5d95

### After - using `primitiveColors.css`

https://github.com/user-attachments/assets/60285eba-051e-48f2-bb2c-b23a56dbeaa0

----

### Before - using `shared/colors.scss`

https://github.com/user-attachments/assets/3d9fae0f-4578-4431-9274-2ba1709be302

### After - using `primitiveColors.css`

https://github.com/user-attachments/assets/546565a1-aa3a-4ff4-a9b8-fb86d4c5bac1

----

### Before - using `shared/colors.scss`

https://github.com/user-attachments/assets/8f23c93a-fcac-463d-ba53-6883690e7676

### After - using `primitiveColors.css`

https://github.com/user-attachments/assets/09f9c18c-6783-4a77-b61e-d4edd795c9d0

----

### Before - using `shared/colors.scss`

https://github.com/user-attachments/assets/d8cb66f7-8828-4003-a9c2-326ea5985a34

### After - using `primitiveColors.css`

https://github.com/user-attachments/assets/fff634d4-a83b-4c00-a5a8-e54451dade47

### CODE REVIEW

**Comment by kelbyhawn** [comment]
@levadadenys it might be worth checking this one out locally too since it's more complicated.

File: apps/src/componentLibrary/dropdown/simpleDropdown/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.8.1](https://github.com/code-dot-org/code-dot-org/pull/62917)
```
Comment (levadadenys) [inline apps/src/componentLibrary/dropdown/simpleDropdown/CHANGELOG.md:5]: Simple dropdown is not updated by this PR.
Styles for simpleDropdown are specified in `apps/src/componentLibrary/dropdown/simpleDropdown/simpleDropdown.module.scss`

Comment (kelbyhawn) [reply apps/src/componentLibrary/dropdown/simpleDropdown/CHANGELOG.md:5]: Ah, thank you! I'll update this.

Comment (kelbyhawn) [reply apps/src/componentLibrary/dropdown/simpleDropdown/CHANGELOG.md:5]: This is updated now! I also updated the "After" video in the PR description.

**Review by levadadenys — APPROVED** [review]
LGTM! Thank you! 🚀

**Pull request merged**